### PR TITLE
Migration of module l10n_be_antibiotic_tax, l10n_be_apb_tax, l10n_be_eco_tax. I created a single commit for these three modules because it's only data. There no code in these modules.

### PR DESCRIPTION
**Pull request by sylvainvh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ Available addons
 addon | version | summary
 --- | --- | ---
 [l10n_be_mis_reports](l10n_be_mis_reports/) | 10.0.1.0.0 | MIS Builder templates for the Belgium P&L and Balance Sheets
+[l10n_be_antibiotic_tax](l10n_be_antibiotic_tax) | 10.0.1.0.0 | Data module to support antibiotics taxes
+[l10b_be_apb_tax](l10n_be_apb_tax) | 10.0.1.0.0 | Data module to support APB taxes
+[l10b_be_eco_tax](l10n_be_eco_tax) | 10.0.1.0.0 | Data module to support BEBAT and RECUPEL taxes
```
Comment (yvaucher) [inline README.md]: Those changes are automatically made by @OCA-git-bot

File: l10n_be_antibiotic_tax/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline l10n_be_antibiotic_tax/__manifest__.py]: Please use short headers https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L1-L3

File: l10n_be_antibiotic_tax/README.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+l10n_be Antibiotic Taxes
```
Comment (yvaucher) [inline l10n_be_antibiotic_tax/README.rst]: s/l10n_be/Belgium ?

File: l10n_be_apb_tax/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline l10n_be_apb_tax/__manifest__.py]: Use short header:
https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L1-L3

File: l10n_be_eco_tax/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline l10n_be_eco_tax/__manifest__.py]: Use short header

File: l10n_be_eco_tax/README.rst [hunk 6]
```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+l10n_be ECO Taxes
```
Comment (yvaucher) [inline l10n_be_eco_tax/README.rst]: s/l10n_be/Belgium

**Review by yvaucher — APPROVED** [review]
Some nitpicking otherwise LGTM

**Comment by sylvainvh** [comment]
@yvaucher Done !

**Comment by sylvainvh** [comment]
There is no "python code" in these modules. I think it why the coverage test doesn't work.

**Review by sbidoul — APPROVED** [review]

**Comment by sbidoul** [comment]
Thanks!

**Comment by sylvainvh** [comment]
You're welcome

**Pull request merged**